tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -265,4 +266,47 @@ public void testUnixSocket() throws Exception
             Files.deleteIfExists(sockFile);
         }
     }
+
+    @Test
+    public void testLog4j2ModuleWithSimpleWebAppWithJSP() throws Exception
+    {
+        Path jettyBase = Files.createTempDirectory( "jetty_base");
+        String jettyVersion = System.getProperty("jettyVersion");
+        DistributionTester distribution = DistributionTester.Builder.newInstance()
+            .jettyVersion(jettyVersion)
+            .jettyBase(jettyBase)
+            .mavenLocalRepository(System.getProperty("mavenRepoPath"))
+            .build();
+
+        String[] args1 = {
+            "--create-startd",
+            "--approve-all-licenses",
+            "--add-to-start=resources,server,http,webapp,deploy,jsp,servlet,servlets,logging-log4j2"
+        };
+        try (DistributionTester.Run run1 = distribution.start(args1))
+        {
+            assertTrue(run1.awaitFor(5, TimeUnit.SECONDS));
+            assertEquals(0, run1.getExitValue());
+            assertTrue(Paths.get(jettyBase.toString(), "resources", "log4j2.xml" ).toFile().exists());
```

Comment:
Cool. this is what I had in mind.

One niggle.  why can't you use ...

``` java
assertTrue(Files.exists(jettyBase.resolve("resources/log4j2.xml")));
```

That would also be compatible with the File Systems present on Unix, OSX, and Windows automatically.
The less back and forth translation between `Path` and `String` the better.